recipes/librdkafka/all/patches/0002-Conanize-cmake-links-1-3-0.patch

```diff
@@ -0,0 +1,38 @@
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -171,14 +171,11 @@ if(WITH_HDRHISTOGRAM)
+ endif()
+
+ if(WITH_ZLIB)
+-  find_package(ZLIB REQUIRED)
+-  target_link_libraries(rdkafka PUBLIC ZLIB::ZLIB)
++  target_link_libraries(rdkafka PUBLIC CONAN_PKG::zlib)
```

Comment:
```suggestion
+  find_package(ZLIB REQUIRED)
+  target_link_libraries(rdkafka PUBLIC ZLIB::ZLIB)
```

Reply:
Thanks for your example. But does it make any difference?

Reply:
Then you inherit of the module generated by zlib as defined in the recipe of zlib.

Reply:
I think that CONAN_PKG does it automatically. 
There's an example: https://github.com/conan-io/conan-center-index/blob/master/recipes/libkml/all/patches/conanize-cmakelists.patch
That package uses the same idea.